src/navigation-timing.js

```diff
@@ -97,14 +97,28 @@ window.NTAPI = window.NTAPI || {};
             console.log(metrics);
         }
 
-        if (metrics.type && window.NTAPI.PostUrl) {
-            var xmlhttp = new XMLHttpRequest();   // new HttpRequest instance
-            xmlhttp.open('POST', window.NTAPI.PostUrl);
-            xmlhttp.setRequestHeader('Content-Type', 'application/json');
-            xmlhttp.send(JSON.stringify(metrics));
-        } else if (window.debug) {
-            console.log('Navigation Timing Settings Definition Error. Metrics Type:', metrics.type, 'PostUrl:', window.NTAPI.PostUrl);
+        // NOTE: analytics -> {} & performanceMetricsArr -> [] are both initialized global's that should be setup via your client application; their purpose is to interface with Segment.com's analytics.js file that's loaded in on the client app
+        if (analytics && performanceMetricsArr) {
```

Comment:
^ I just thought about this... what do you think about namespacing this "performanceMetricsArr" variable? maybe window.NTAPI.metricsArray ?

(... and also change in a PR for hope as well)